pkg/kopia/command/parse_command_output.go

```diff
@@ -382,3 +382,26 @@ func IsEqualSnapshotCreateStats(a, b *SnapshotCreateStats) bool {
 		a.SizeEstimatedB == b.SizeEstimatedB &&
 		a.ProgressPercent == b.ProgressPercent
 }
+
+var ANSIEscapeCode = regexp.MustCompile(`\x1b[^m]*?m`)
+var kopiaErrorPattern = regexp.MustCompile(`ERROR\s+(.*)`)
+
+// ErrorFromOutput parses the output of a kopia and returns an error, if found
+func ErrorsFromOutput(output string) []error {
```

Comment:
Is this function only for `kopia repository` sub commands and not for other commands?
For example for `kopia snapshot restore` commands, it could be of format:
```
error restoring: restore error: error copying: copy file: error creating file:
```

Another example could be:
```
error setting attributes: could not change owner/group
```

Reply:
Thanks, I added couple more cases. I went through Kopia's repo and found that they have several unstructured error logs, but they will be covered by generic "last x log lines"  mechanism.